sw/simulator/nps/nps_fdm_gazebo.cpp

```diff
@@ -83,6 +85,35 @@ static struct gazebocam_t gazebo_cams[VIDEO_THREAD_MAX_CAMERAS] =
 { { NULL, 0 } };
 #endif
 
+#ifdef NPS_SIMULATE_RANGE_SENSORS
+#include "subsystems/abi.h"
+static void gazebo_init_range_sensors(void);
+static void gazebo_read_range_sensors(void);
+#ifndef NPS_GAZEBO_RANGE_MAX_SENSORS
+#define NPS_GAZEBO_RANGE_MAX_SENSORS 255
+#endif
+
+gazebo::sensors::RaySensorPtr RaySensorPtr_array[NPS_GAZEBO_RANGE_MAX_SENSORS];
+
+
+#ifndef NPS_GAZEBO_RANGE_AMOUNT_SENSORS
+#define NPS_GAZEBO_RANGE_AMOUNT_SENSORS 0
+#endif
+
+
+#ifndef NPS_GAZEBO_RANGE_ORIENTATION
+#define NPS_GAZEBO_RANGE_ORIENTATION 0, 0, 0
+#endif
```

Comment:
This should not be used but rather use the REMOTE_GROUND_ORIENTATIONS, this whay simulation and reality are the same.